src/components/shared/PageContainer.tsx

```diff
@@ -56,11 +56,16 @@ function PageContainer({ children, hideBackground, pageTitleProps }: Props) {
         >
             {notification ? (
                 <Snackbar
+                    anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
                     open={displayAlert}
-                    autoHideDuration={5000}
+                    autoHideDuration={7500}
```

Comment:
Making the notification show for just a bit longer. Since 7.5 seconds could be long added support to allow closing notifications.